### PR TITLE
Sets author to campaign run nodes

### LINKED ISSUE #5961: Create campaign runs in a script
Create campaign run nodes for all open campaigns (published or unpublished)

### PR DESCRIPTION
**Pull request by chloealee**
#### What does this PR do?

Sets author to campaign run nodes. 
#### What are the relevant tickets?

Nodes created in https://github.com/DoSomething/phoenix/pull/6018

fixes #5961 with the help of #6016

### CODE REVIEW

**Comment by chloealee** [comment]
@angaither I should probably wait to merge this until #6016 is merged and we deploy again with #6018 code, right? I think that's why there are conflicts?

**Comment by angaither** [comment]
@chloealee you could rebase this with dev and recommit so the entire file is included

**Comment by chloealee** [comment]
got it, thanks @angaither! Just rebased.

File: lib/modules/dosomething/dosomething_campaign_run/dosomething_campaign_run.install [hunk 1]
```diff
@@ -20,3 +20,57 @@ function dosomething_campaign_run_update_7002(&$sanebox) {
...
+    $created_dates = explode(',', $active_campaign->created_dates);
+
+    $campaign_run_node->title = $active_campaign->title_field_value . ' run 1';
+    $campaign_run_node->field_campaigns[$active_campaign->src_lang][]['target_id'] = $active_campaign->entity_id;
+
+    $php_timestamp_date = date('y/m/d', $created_dates[0]);
+    $campaign_run_node->field_run_date[$active_campaign->src_lang][]['value'] = $php_timestamp_date;
+
+    $campaign_run_node->author = 'admin@dosomething.org';
+    $campaign_run_node->uid = '343';
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_campaign_run/dosomething_campaign_run.install]: did you test this? I don't think this works?

**Comment by chloealee** [comment]
@angaither everything is working now! One clarifying question I had - right now it's hard coded that you're the author. Do you want me to automatically pull the author or have this remain as is?

**Comment by angaither** [comment]
this looks :ok: the reason I mentioned it could be my uid is because I it's the same user in all the environments

**Comment by angaither** [comment]
:+1:

**Pull request merged**